src/store/settings/settings-state.ts

```diff
@@ -22,7 +23,7 @@ export interface SettingsState {
   isShowLoader: boolean;
   isManualBackupMade: boolean;
   isCloudBackupMade: boolean;
-  isOnRampPossibility: boolean;
+  onRampOverlayState: OnRampOverlayState;
   isOnRampHasBeenShownBefore: boolean;
```

Comment:
here, just like `isEnableAdsBanner`, so we know that value with this naming can't be used in the future